crates/re_viewer/src/ui/data_ui/mod.rs

```diff
@@ -25,6 +25,7 @@ pub enum UiVerbosity {
     Small,
 
     /// At most this height
+    #[allow(dead_code)]
```

Comment:
It's still checked in quite a few places, but never constructed... I'm keeping it around as I imagine it's gonna see new usage pretty soon?

Reply:
I vote to get rid of it